0_download_data/0.download_NF1_data.yml

```
@@ -0,0 +1,28 @@
+name: download-NF1-data
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.8
+  - conda-forge::scikit-image=0.19.2
+  - conda-forge::pandas=1.4.4
```

Consider unpinning from static/precise versions of certain libraries unless you have a specific need (for ex. Pandas is currently at 1.5.2). If there is a specific need, I'd recommend providing a comment (for ex. `# using pandas 1.4.4 to avoid <specific conflicts>`) Unpinning will oftentimes automatically enable performance increases or address important issues like security or accuracy which are otherwise difficult to keep up with manually. Using a `>=`  if you wanted to be certain that versions were above a specific threshold could be helpful.  For ex:

```suggestion
  - conda-forge::pandas>=1.4.4
```

---

Very interesting! 

@d33bs Can you provide what packages you think would be less likely to cause issues if the version was upgraded?

---

This comment came as a general recommendation; I'm not certain of specific benefits with each dependency listed. Also consider whether and how much your environment files differ from one another (for example, `CellProfiler_pipelines/download_cellprofiler.yml`). Minor differences with dependencies that treat shared data could lead to unintended discrepancies in the results.

---

Pandas seems to be the only discrepancy so I will upgrade the pandas in this environment since pandas in the cellprofiler pipelines environment does not come with a version specification. 

@d33bs I think that seems fair!